### PR TITLE
Car brands

### PR DESCRIPTION
**Pull request by bltemrah**
The test scenarios are for below car brands:
Audi
BWM
Tofas

### CODE REVIEW

File: src/main/java/org/example/sample-1.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Car Brand
+Audi
```
Comment (AlexandruSendrea) [inline src/main/java/org/example/sample-1.txt:2]: @bltemrah  As I remember that the ticket says the first brand should be Mercedes

Comment (betulperco) [inline src/main/java/org/example/sample-1.txt:2]: its fine

File: src/main/java/org/example/sample-1.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+BMW
+Tofas
```
Comment (betulperco) [inline src/main/java/org/example/sample-1.txt:4]: Please add Mercedes also

File: src/main/java/org/example/sample-1.txt (hunk 1 above)
Comment (bltemrah) [reply src/main/java/org/example/sample-1.txt]: @AlexandruSendrea good spot, I updated it.

File: src/main/java/org/example/sample-1.txt (hunk 2 above)
Comment (bltemrah) [reply src/main/java/org/example/sample-1.txt]: thanks, I updated it.

**Pull request merged**